<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoreDirs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionSwitch_View"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSwitch_View"/>
   <addaction name="actionCascade"/>
   <addaction name="actionTileWindow"/>
   <addaction name="actionTile2Window"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionCloseTab">
   <property name="text">
    <string>Close Tab Page</string>
   </property>
   <property name="toolTip">
    <string>Close tab</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCloseOtherTab">
   <property name="text">
    <string>Close Other Tab</string>
   </property>
   <property name="toolTip">
    <string>Close Other Tab</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCloseAllTab">
   <property name="text">
    <string>Close All Pages</string>
   </property>
   <property name="toolTip">
    <string>Close All Pages</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSwitch_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons8-tab-24.png</normaloff>:/icons/icons8-tab-24.png</iconset>
   </property>
   <property name="text">
    <string>ShowTab</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCascade">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/400.bmp</normaloff>
     <disabledoff>:/new/icons/icons/401.bmp</disabledoff>:/new/icons/icons/400.bmp</iconset>
   </property>
   <property name="text">
    <string>Cascade</string>
   </property>
   <property name="toolTip">
    <string>Cascade windows</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionTileWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/406.bmp</normaloff>
     <disabledoff>:/new/icons/icons/407.bmp</disabledoff>:/new/icons/icons/406.bmp</iconset>
   </property>
   <property name="text">
    <string>TileWindow</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionTile2Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/icons/404.bmp</normaloff>
     <disabledoff>:/new/icons/icons/405.bmp</disabledoff>:/new/icons/icons/404.bmp</iconset>
   </property>
   <property name="text">
    <string>Tile2Window</string>
   </property>
   <property name="toolTip">
    <string>Tile 2 Windows</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
